.circleci/release.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: release.giantswarm.io/v1alpha1
 kind: Release
 metadata:
   annotations:
-    giantswarm.io/docs: https://pkg.go.dev/github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1?tab=doc#Release
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
```

Comment:
This was made public today so I just wanted to update to the correct URL.

Reply:
Nice :heart:

Reply:
+1 ❤️